src/agent/coverage/src/record/linux/debugger.rs

```diff
@@ -46,7 +49,30 @@ impl<'eh> Debugger<'eh> {
             return Err(err);
         }
 
-        let output = child.wait_with_output()?;
+        // Currently unavailable on Linux.
+        let status = None;
```

Comment:
If we had the `wait(2)` status integer that `pete` internally parses, then we could construct a standard library `ExitStatus` using [`ExitStatusExt::from_raw()`](https://doc.rust-lang.org/std/process/struct.ExitStatus.html#method.from_raw).

This raw `wait(2)` status isn't currently exposed in the `Stop` enum. Rather than try to be clever and resynthesize one from the exited/signaled/core dumped info we _do_ get, just omit it for now.

Reply:
We should create an issue in pete and link to it from here so we don't lose track of this